lib/validators/options_processor/teams.js

```diff
@@ -19,6 +19,26 @@ class Teams {
     }
     return _.uniq(teamMembers)
   }
+
+  static async extractTeamMemberships (context, teams, users) {
+    const teamMembers = []
+    let membershipState = false
+
+    if (!teams || teams.length === 0) return teamMembers
+
+    for (let user of users) {
+      for (let team of teams) {
+        try {
+          membershipState = await getTeamMembershipState(context, team, user)
+        } catch (err) {
+          throw err
+        }
+
+        if (membershipState) { teamMembers.push(user) }
+      }
+    }
+    return _.uniq(teamMembers)
+  }
 }
 
 const getTeamMembers = async (context, team) => {
```

Comment:
i see that `[owner](https://github.com/mergeability/mergeable/blob/master/lib/validators/options_processor/owners.js#L37)` file still uses this function, is there a particular reason for it?